### PR TITLE
Code quality check

### PR DESCRIPTION
**Pull request by ductiletoaster**

### CODE REVIEW

File: packages/pixeloven-cli/src/index.test.ts [hunk 1]
```diff
@@ -8,13 +14,38 @@ const caller = () => {
...
-            const cliSpy = jest.spyOn(cli, "default");
+            mainSpy.expects("default").once();
             caller();
-            expect(cliSpy).toHaveBeenCalledTimes(1);
+            mainSpy.verify();
         });
     });
 });
+
+// import "jest";
```
Comment (jimmynotjim) [inline packages/pixeloven-cli/src/index.test.ts]: Looks like this is all duplication, can it be removed?

File: packages/pixeloven-cli/src/commands/delete.ts [hunk 2]
```diff
@@ -1,9 +1,9 @@
-import { PixelOvenRunContext } from "../types";
+import { GluegunRunContext } from "gluegun";
```
Comment (jimmynotjim) [inline packages/pixeloven-cli/src/commands/delete.ts]: It's happening! 🙌

File: packages/pixeloven-cli/src/index.test.ts (hunk 1 above)
Comment (ductiletoaster) [reply packages/pixeloven-cli/src/index.test.ts]: This isn't ready for PR haha! I usually create a PR with no reviewers so I can track my changes more easily.

Comment (jimmynotjim) [reply packages/pixeloven-cli/src/index.test.ts]: whoops, sorry. Lemme know when you want a second look at it.

**Pull request merged**